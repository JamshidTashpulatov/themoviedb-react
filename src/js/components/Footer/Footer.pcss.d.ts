export const footer: string;
export const tmdb: string;
export const github: string;
