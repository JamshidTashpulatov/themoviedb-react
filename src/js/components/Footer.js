import React from 'react';
import css from '../../css/footer.css';

const tmdbLogo = require('../../img/408x161-powered-by-rectangle-blue.png');
const Footer = () => (
  <div className="footer section section__dark">
    <div className="container">
      <img className="footer--logo" src={tmdbLogo} />
      <a href="https://github.com/lviit/themoviedb-react/">https://github.com/lviit/themoviedb-react/</a>
    </div>
  </div>
);

export default Footer
