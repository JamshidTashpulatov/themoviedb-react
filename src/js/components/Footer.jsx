import React from 'react';
import Styles from '../../css/footer.pcss';

const tmdbLogo = require('../../img/408x161-powered-by-rectangle-green.png');

const Footer = () => (
  <div className="footer section section__dark">
    <div className="container">
      <img className={Styles.logo} src={tmdbLogo} alt="" />
      <a href="https://github.com/lviit/themoviedb-react/">https://github.com/lviit/themoviedb-react/</a>
    </div>
  </div>
);

export default Footer;
