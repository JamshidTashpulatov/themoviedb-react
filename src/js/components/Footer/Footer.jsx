import React from "react";
import Styles from "./footer.pcss";

const tmdbLogo = require("../../../img/408x161-powered-by-rectangle-green.png");

const Footer = () => (
  <div className={Styles.footer}>
    <img className={Styles.logo} src={tmdbLogo} alt="" />
    <a href="https://github.com/lviit/themoviedb-react/">
      https://github.com/lviit/themoviedb-react/
    </a>
  </div>
);

export default Footer;
