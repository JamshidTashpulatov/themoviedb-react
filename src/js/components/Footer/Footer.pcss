@import "../../../css/variables.pcss";

.footer {
  position: fixed;
  bottom: 0;
  background: var(--bgalt);
  width: 100%;
  display: flex;
  padding: 7px 0 0;
  justify-content: center;
  align-items: center;

  @media (--desktop) {
    padding: 10px 0 5;
  }
}

.tmdb {
  display: block;
  width: 70px;
  margin-right: 20px;

  @media (--desktop) {
    width: 100px;
  }
}

.github {
  display: block;
  width: 60px;

  @media (--desktop) {
    width: 80px;
  }
}
