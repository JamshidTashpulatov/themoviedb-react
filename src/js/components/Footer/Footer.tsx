import * as React from "react";
import * as Styles from "./Footer.pcss";
import tmdbSvg from "../../../img/tmdb.svg";
import githubSvg from "../../../img/github.svg";

const Footer = () => (
  <div className={Styles.footer}>
    <div
      className={Styles.tmdb}
      dangerouslySetInnerHTML={{ __html: tmdbSvg }}
    />
    <a
      className={Styles.github}
      href="https://github.com/lviit/themoviedb-react/"
      dangerouslySetInnerHTML={{ __html: githubSvg }}
    />
  </div>
);
export default Footer;
