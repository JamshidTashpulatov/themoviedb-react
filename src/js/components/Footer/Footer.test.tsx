import * as React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

it("renders correctly", () => {
  const tree = shallow(<Footer />);

  expect(tree).toMatchSnapshot();
});
