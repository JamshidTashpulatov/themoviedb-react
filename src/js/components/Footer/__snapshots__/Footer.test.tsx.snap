// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="footer"
>
  <img
    alt="tmdb"
    className="tmdb"
  />
  <a
    href="https://github.com/lviit/themoviedb-react/"
  >
    <img
      alt="github"
      className="github"
    />
  </a>
</div>
`;
